src/harness/reference_models/interference/aggregate_interference.py

```diff
@@ -0,0 +1,366 @@
+  neighborhood of the FSS co-channel protection entity.
+
+  Args:
+    fss_record: FSS co-channel protection entity
+    cbsd_list: list of CBSD objects containing registrations and grants
+  Returns:
+    Aggregate interference to FSS co-channel in the nested dictionary format.
+    {latitude : {longitude: [aggr_interf1(mW), aggr_interf2(mW)]}}
+  """
+  grant_list = interf.getAllGrantInformationFromCbsdDataDump(cbsd_list)
+
+  # Get the protection point of the FSS
+  fss_point = fss_record['record']['deploymentParam'][0]['installationParam']
+  protection_point = (fss_point['latitude'], fss_point['longitude'])
+
+  # Get the frequency range of the FSS
+  fss_freq_range = fss_record['record']['deploymentParam'][0]\
+                     ['operationParam']['operationFrequencyRange']
+  fss_low_freq = fss_freq_range['lowFrequency']
+
+  # Get FSS information
+  fss_info = interf.FssProtectionPoint(latitude=fss_point['latitude'], 
+               longitude=fss_point['longitude'],
+               height_agl=fss_point['height'],
+               max_gain_dbi=fss_point['antennaGain'],
+               pointing_azimuth=fss_point['antennaAzimuth'],
+               pointing_elevation=fss_point['antennaDowntilt'])
+
+  # Get channels for co-channel CBSDs
+  protection_channels = interf.getProtectedChannels(fss_low_freq, interf.CBRS_HIGH_FREQ_HZ)
+
+  aggregate_interference.__init__()
+
+  aggregateInterferenceForPoint(protection_point, 
+    protection_channels, fss_low_freq, 
+    interf.CBRS_HIGH_FREQ_HZ, grant_list, fss_info, 
+    None, interf.ProtectedEntityType.FSS_CO_CHANNEL)
+
+  return aggregate_interference.GetAggregateInterferenceInfo() 
+
+def calculateAggregateInterferenceForFssBlocking(fss_record, cbsd_list):
+  """Calculates aggregate interference for FSS blocking.
+  
+  Calculates aggregate interference for FSS blocking pass band based on the 
+  interferences caused by CBSD's grants in the neighborhood of the FSS 
+  blocking protection entity.
+
+  Args:
+    fss_record: FSS blocking protection entity
+    cbsd_list: list of CBSD objects containing registrations and grants
+  Returns:
+    Aggregate interference to FSS blocking in the nested dictionary format.
+    {latitude : {longitude: [aggr_interf1(mW), aggr_interf2(mW)]}}
+  """
+  grant_list = interf.getAllGrantInformationFromCbsdDataDump(cbsd_list)
+
+  fss_ttc_flag = fss_record['ttc']
+  # Get the protection point of the FSS
+  fss_point = fss_record['record']['deploymentParam'][0]['installationParam']
+  protection_point = (fss_point['latitude'], fss_point['longitude'])
+
+  # Get the frequency range of the FSS
+  fss_freq_range = fss_record['record']['deploymentParam'][0]\
+                     ['operationParam']['operationFrequencyRange']
+  fss_low_freq = fss_freq_range['lowFrequency']
+  fss_high_freq = fss_freq_range['highFrequency']
+
+  # Get FSS information
+  fss_info = interf.FssProtectionPoint(latitude=fss_point['latitude'], 
```

Comment:
This object defined in interference.py, is really a FssInfo (like the EscInfo) rather than a FssProtectionPoint, and things like latitude and longitude are not used and could be removed. 
Otherwise we get confused between these info containing position and the protection_point also containing the real location.

Reply:
Removed   latitude, longitude from FssProtectionPoint.    Also renamed FssProtectionPoint to FssInformation